.circleci/config.yml

```diff
@@ -0,0 +1,10 @@
+version: 2 # use CircleCI 2.0 
+jobs: # a collection of steps
+  build: # runs not using Workflows must have a `build` job as entry point
+  
+    docker: # run the steps with Docker
+      - image: circleci/openjdk:8-jdk-stretch # ...with this image as the primary container; this is where all `steps` will run
+      
+    steps: # a collection of executable commands
+      - checkout
+      - run: mvn clean compile
```

Comment:
change this to `./mvnw clean build`

Also learn why we need to do this.

Reply:
done